examples/resources/pingfederate_oauth_idp_adapter_mapping/resource.tf

```diff
@@ -0,0 +1,17 @@
+resource "pingfederate_oauth_idp_adapter_mapping" "oauthIdpAdapterMapping" {
```

Comment:
I'm unsure if this example is very good - I didn't find any examples of this resource in facile or in a sample PF data.json